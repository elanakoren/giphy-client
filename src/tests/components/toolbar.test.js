// @flow
import React from 'react';
import Toolbar from '../../components/toolbar';
import {shallow} from 'enzyme';

describe('gifList', () => {
    it('renders correctly', () => {
        const fakeGifActions = {firstFn: () => {}};
        const wrapper = shallow(<Toolbar {...{gifActions: fakeGifActions}}/>);
        expect(wrapper).toMatchSnapshot();
    });
});