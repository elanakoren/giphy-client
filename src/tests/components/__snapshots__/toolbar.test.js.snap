// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`gifList renders correctly 1`] = `
<div
  className="toolbar-grid"
>
  <Search
    gifActions={
      Object {
        "firstFn": [Function],
      }
    }
  />
  <SortBar
    gifActions={
      Object {
        "firstFn": [Function],
      }
    }
  />
</div>
`;
